phenopacketlab-restapi/src/main/java/org/monarchinitiative/phenopacketlab/restapi/controller/ValidateController.java

```diff
@@ -0,0 +1,26 @@
+package org.monarchinitiative.phenopacketlab.restapi.controller;
+
+import org.monarchinitiative.phenopacketlab.core.ValidateService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(value = "${api.version}/validate", method = RequestMethod.POST)
+public class ValidateController {
+
+    private final ValidateService validateService;
+
+    public ValidateController(ValidateService validateService) {
+        this.validateService = validateService;
+    }
+
+    @PostMapping
+    public ResponseEntity<String> validatePhenopacket(@RequestBody String phenopacketString) {
```

Comment:
I think we may need more detail here, not just one `String`. I think I'll need to do the same trick in Phenopacket tools with `jackson-annotations` in `provided` scope to implement automaitc JSON serialization of the validation results.

Reply:
what type of extra detail do you think we would need? should the validate endpoint take more than just a phenopacket string?

Reply:
Sorry, I meant output. The validation results can be more granular.

Reply:
We are also trying to make an API standard of things that should always be in a response. Could bring this up in that meeting.